README.md

```diff
@@ -1,2 +1,54 @@
 # Awesome-books
 A webapp to save a list of books.
+
+## Built With
+
+- HTML, CSS
+- Webhint, Stylelint, Github Actions
+
+## Live Demo
+
+[https://nayankaran.github.io/Awesome-books/](https://nayankaran.github.io/Awesome-books/)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- Git
+- Code Editor. e.g., VS Code
+
+### Setup
+
+- Clone the repository using the `git clone` command.
+
+### Usage
+
+- Open the index.html file.
+
+## Author
+
+👤 **Debabrata Mandal (Nayan Karan)**
+
+- GitHub: [@NayanKaran](https://github.com/NayanKaran)
+- Twitter: [@bapidebabrata](https://twitter.com/bapidebabrata)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/debabrata-mandal-83461696/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```

Comment:
- [ ] [Optional] It would be nice adding a link to your issue page so that visitors to your projects can make contributions to the project.

Reply:
@danmainah The link to issue page is already there. Can you please clarify it in details?